guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc

```diff
@@ -256,6 +256,107 @@ ifeval::["{build}" == "satellite"]
+
+You can provision hosts from {Project} using the UEFI HTTP Boot.
+This is the only method with which you can provision hosts in IPv6 network.
+
+.Procedure
+
+. On {Project}, enable the `foreman-proxy-http` and `foreman-proxy-httpboot` plug-ins. Disable system checks with the `--disable-system-checks`  and `--skip-checks-i-know-better` options.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {installer-scenario} \
+--foreman-proxy-httpboot true \
+--foreman-proxy-http true \
+--disable-system-checks \
+--skip-checks-i-know-better
+----
+
+. In the {Project} web UI, navigate to *Hosts* > *Create Host*.
+. In the *Name* field, enter a name for the host.
+. Click the *Organization* and *Location* tabs and change the context to match your requirements.
+. From the *Host Group* list, select a host group that you want to use to populate the form.
+. Click the *Interface* tab, and on the host's interface, click *Edit*.
+. Verify that the fields are populated with values. Note in particular:
++
+* The *Name* from the *Host* tab becomes the *DNS name*.
+* {ProjectServer} automatically assigns an IP address for the new host.
++
+. In the *MAC address* field, enter a MAC address of the host's provisioning interface. This ensures the identification of the host during the PXE boot process.
+. Ensure that {ProjectServer} automatically selects the *Managed*, *Primary*, and *Provision* options for the first interface on the host. If not, select them.
+. Click *OK* to save. To add another interface, click *Add Interface*. You can select only one interface for *Provision* and *Primary*.
+. Click the *Operating System* tab, and verify that all fields contain values. Confirm each aspect of the operating system.
+. From the *PXE Loader* list, select *Grub2 UEFI HTTP*.
+. Optional: Click *Resolve* in *Provisioning template* to check the new host can identify the right provisioning templates to use.
++
+For more information about associating provisioning templates, see xref:Configuring_Provisioning_Resources-Creating_Provisioning_Templates[].
+ifeval::["{build}" == "satellite"]
+. Click the *Parameters* tab, and ensure that a parameter exists that provides an activation key. If not, add an activation key.
+endif::[]
+ifeval::["{build}" == "foreman"]
+. If you use the Katello plugin, click the *Parameters* tab, and ensure that a parameter exists that provides an activation key. If not, add an activation key.
+endif::[]
+. Click *Submit* to save the host details.
++
+For more information about network interfaces, see {BaseURL}/managing_hosts/index#Adding_Network_Interfaces[Adding network interfaces].
+. Start the host and set its boot mode to UEFI HTTP.
+. Select the *Kickstart default PXEGrub2 HTTP Boot*. Until BZ#[BZ#] is resolved, selecting *Kickstart Default PXEGrub2* option fails the provisioning process. Until BZ#[] is resolved, ignore the *error: unrecognized network address* warnings that are displayed during boot. 
```

Comment:
Hi @lzap, I have created a draft procedure for HTTP BOOT Provisioning in this PR. I heard in your presentation during the [community demo](https://www.youtube.com/watch?v=dHfU31ZmPDs&feature=emb_logo) about these two bugs that you have submitted and I want to note them in docs. Can you share bug numbers?

Reply:
By the time this is released, Grub2 should have been fixed already, so I would: Select Kickstart default PXEGrub2. If you encounter issues with grub2 loading kernel, you are using old version of grub2. Update grub2-efi package and rerun the installer to copy `grubx64.efi` to the TFTP/HTTP boot directory. In the meantime, you can use "Kickstart default PXEGrub2 HTTP Boot" option to workaround the problem.

The bug relevant to this is https://bugzilla.redhat.com/show_bug.cgi?id=1811561

Reply:
Cool, thanks for providing a workaround, I'll add it to docs, but I have questions about it:
Where do you need to update the grub2-efi package? Maybe it is a good idea to add a step at the beginning of the procedure to update this package to the latest version on Satellite? If the package is of the latest version, no workaround is needed.

Reply:
Definitely this is worth actually a module we can include at various places.

Basically:

1. Install or upgrade package named `grub2-efi-x64`.
1. Run foreman-installer. The installer will copy file `/boot/efi/EFI/*/grubx64.efi` into `/var/lib/tftproot/grub2` overwriting the previous file.

Reply:
The steps to update `grub2-efi` and run the installer after are in the procedure.